frontend/app/src/stores/useRoomStore.ts

```diff
@@ -0,0 +1,11 @@
+import { create } from "zustand";
+
+interface RoomState {
+    roomId: string;
+    modifyRoomId: (value: string) => void;
+}
+
+export const useRoomStore = create<RoomState>(set => ({
```

Comment:
is the store still being used? from my memory we agreed it's not necessary anymore

Reply:
Remove zustand store in [a8a542f](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/18/commits/a8a542fa08caee9779d125e972f19d7565ba9dd0)